### DISCUSSION TITLE
does copy not work with tiling window managers?

### DISCUSSION THREAD

**Discussion by txtyash** [flameshot-org/flameshot#2981] [General]
The copy option has never worked for me. Not in xmonad and not in awesome wm.
```
❯ flameshot full -c -p ~/screenshots_full
flameshot: info: Capture saved as /home/zim/screenshots_full/2022-11-13_14-29.png

~ via 🌙 v5.4.4 via  v19.0.1
❯ flameshot full  -p ~/screenshots_full -c
flameshot: info: Capture saved as /home/zim/screenshots_full/2022-11-13_14-30.png
```
Neither inside the terminal nor as a binding

**Comment by mmahmoudian** [comment]
I cannot help you when you have not provided any information about the setup! Flameshot version number, how you have installed it, what is your OS, what is your clipboard manager and etc. On top of all these, Please search in [the issues](https://github.com/flameshot-org/flameshot/issues) and if your issue is not listed, please fill a complete bug report. Discussions section is, as the name suggests, for discussions and not solving issues. Thanks.
